packages/kokoas-client/src/pages/projEstimate/estimateDataGrid/EstimateDataGridContainer.tsx

```diff
@@ -20,7 +20,7 @@ export const EstimateDataGridContainer = ({
       spacing={1}
       sx={{
         maxWidth: `calc(100vw - ${menuOpen ? menuWidth + menuOffsetWidth : menuOffsetWidth}px)`, 
-        height: 'calc(100vh - 180px)',
+        height: 'calc(100vh - 560px)',
```

Comment:
```suggestion
        height: 'calc(100vh - 300px)',
```
見積行が私のブラウザで2行しか見えなくなってしまっていました。
こちらで確認した限りではこのくらいが妥当そうでしたが、確認をお願いいたします。

Reply:
![スクリーンショット 2024-06-17 101022](https://github.com/cocosumo/yumecoco-monorepo/assets/87287796/dfcf77a7-9b58-4687-be25-f1a2994f73bb)
![スクリーンショット 2024-06-17 100824](https://github.com/cocosumo/yumecoco-monorepo/assets/87287796/0dc818cb-6073-4e4b-8c14-af79388f0d77)

ノートPCの画面（上）と、別ディスプレイの画面（下）の両方送ります。
解像度は、(上)1366 × 768、(下)1920 × 1080です。

使用対象者でノートPCだけを使用している方も数人はいそうです。

Reply:
なるほど。ありがとうございます！全然異なりますので、早川さんの修正提案をそのまま適用させていただきました。

画像に利益率がありますので、削除OKです。

Reply:
ホントですね、失礼しました。
とりあえず画像だけ削除しておきました。